libraries/helpers.rb

```diff
@@ -0,0 +1,140 @@
+            centos-ansible
+            centos-ansible-testing
+            centos-ansible-debuginfo
+            centos-ceph
+            centos-ceph-testing
+            centos-gluster
+            centos-gluster-testing
+            centos-hyperscale
+            centos-hyperscale-experimental
+            centos-hyperscale-hotfixes
+            centos-hyperscale-spin
+            centos-nfs-ganesha
+            centos-nfs-ganesha-testing
+            centos-nfv-extras
+            centos-nfv-extras-testing
+            centos-nfv-extras-debuginfo
+            centos-openstack
+            centos-openstack-testing
+            centos-openstack-debuginfo
+            centos-opstools
+            centos-opstools-testing
+            centos-ovirt
+            centos-ovirt-testing
+            centos-ovirt-debuginfo
+            centos-qpid-proton
+            centos-qpid-proton-testing
+            centos-qpid-proton-debuginfo
+            centos-rabbitmq
+            centos-rabbitmq-testing
+            centos-rabbitmq-debuginfo
+            centos-samba
+            centos-samba-testing
+          )
+        else
+          %w(
+            base
+            appstream
+            powertools
+            extras
+            centosplus
+            fasttrack
+            debuginfo
+            cr
+            highavailability
+            centos-advanced-virtualization
+            centos-advanced-virtualization-testing
+            centos-advanced-virtualization-debuginfo
+            centos-ansible
+            centos-ansible-testing
+            centos-ansible-debuginfo
+            centos-ceph
+            centos-ceph-testing
+            centos-gluster
+            centos-gluster-testing
+            centos-nfs-ganesha
+            centos-nfs-ganesha-testing
+            centos-nfv-extras
+            centos-nfv-extras-testing
+            centos-nfv-extras-debuginfo
+            centos-openstack
+            centos-openstack-testing
+            centos-openstack-debuginfo
+            centos-opstools
+            centos-opstools-testing
+            centos-ovirt
+            centos-ovirt-testing
+            centos-ovirt-debuginfo
+            centos-qpid-proton
+            centos-qpid-proton-testing
+            centos-qpid-proton-debuginfo
+            centos-rabbitmq
+            centos-rabbitmq-testing
+            centos-rabbitmq-debuginfo
+            centos-samba
+            centos-samba-testing
+          )
+        end
+      end
+
+      def centos_openstack_version
+        if yum_centos_stream?
+          'xena'
+        else
+          'victoria'
+        end
+      end
+
+      def centos_vault_version
+        if yum_centos_stream?
+          "#{node['platform_version'].to_i}-stream"
+        else
+          '8.3.2011'
+        end
+      end
+
+      private
+
+      def yum_centos_stream?
+        respond_to?(:centos_stream_platform?) && centos_stream_platform?
+      end
```

Comment:
This does not appear to be backward compatible with https://github.com/sous-chefs/yum-centos/blob/5.2.0/metadata.rb#L9

I have 15 set in my test kitchen, and it borked. Bumping to 16 fixed it.